### ISSUE TITLE
controller recognized in game but i cant play buttons not working (wizard of legend)

### ISSUE THREAD

**Issue by arsen213** [x360ce/x360ce#792]
controller recognized in game but i cant play buttons not working in game

**Comment by jrogge** [comment]
https://gamerinfo.net/game/wizard-of-legend/x360ce/
Have you done everything in this tutorial? I had tried to put the x360ce app in the same directory as the shortcut steam makes but it that's not the actual exe, follow to tutorial to see how to find the exe.

**Comment by arsen213** [comment]
yup still not working and i tried the renaming thing u kno  xinput1_3.dll/xinput1_2.dll etc

**Comment by hunterspy132** [comment]
same issue , any improvement ?

**Comment by Raso93** [comment]
Same issue, I can configure spell buttons but I can't configure the movement ones, pls post if you find a solution, I'll do the same.

**Comment by xborjax** [comment]
I just registered in github to tell you this:
Guys, the solution is download the latest version (1.02c right now).

**Comment by EvoJocys** [comment]
Sorry for the eight years late answer. The project was dormant from 2020 and has been active again since August 2026.

This report is about the 3.x line, where x360ce was an xinput DLL copied into each game's folder. The current 4.x line works differently: it creates a virtual Xbox 360 controller that every game sees, with nothing copied into game folders. Game-specific problems from the DLL era do not carry over.

The latest release is 4.21 (September 2026): https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
